### DISCUSSION TITLE
Restricted When I'm not in a Sanctioned Region

### DISCUSSION THREAD

**Discussion by t0mcr8se** [community/community#118684] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have visited a sanctioned region (my hometown) for spending a holiday with my family and I accessed Github from there and so my github as well as my company's github accounts were restricted. Most of our daily work is done using private repositories and we can't access our work anymore, and we cannot push any new code to these restricted repositories.

Due to the restriction, I had to cancel my family holiday plans and travel to another (non-restricted) country the next day so that I can appeal and contact support to lift this restriction. I also removed myself from my company's github organization so that their work is not affected, but the organization is still restricted.

These restrictions are now unnecessary as I'm not in that region anymore. But I still don't have an answer from github support.

I wonder how other developer platforms (like gitlab) do not restrict accounts in the sanctioned regions but github still does that?

Please take immediate action, this is very time critical and many people are depending on this.

**Comment by t0mcr8se** [comment]
To add to this, I didn't push any commits to any repositories from the sanctioned region

**Comment by ebndev** [comment]
Hi @t0mcr8se,

This community is managed by GitHub employees, but it is not a support forum. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
